tasks/session_management_service.yml

```
@@ -49,6 +49,10 @@
       IpLookupService__ApiToken: "{{ session_management_configuration.ip_stack_api_token }}"
       CloudXRManagement__BaseUrl: "http://{{ container_names.cloudxr_management }}:5000"
       Sentry__Dsn: "{{ session_management_configuration.sentry_dsn }}"
+      AdaptiveInstanceScaling__Enabled: "{{ session_management_configuration.adaptive_instance_scaling.enabled }}"
```

```suggestion
      AdaptiveInstanceScaling__Enabled: "{{ session_management_configuration.adaptive_instance_scaling.enabled | ternary('true', 'false') }}"
```

see https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#defining-different-values-for-true-false-null-ternary